CHANGELOG.md

```diff
@@ -13,6 +16,48 @@
 ### Fixed
 
 ### Security
+-->
+
+## [0.18.8] - 2023-04-27
+
+This release includes several dependency updates, some updates to make existing APIs more friendlier and a change to a
+`Decoder` for extra flexibility.
+
+We have updated the `Retry` API to make it more convenient to override the default timeout in between sleeps. We
+previously had to do something like the following:
+
+```scala
+Retry.retry(maxRetries = 5, inBetweenSleep = Some(2.seconds))(f)
+```
+
+But are now able to provide a duration directly, without wrapping it in an `Option`:
+
+```scala
+Retry.retry(maxRetries = 5, inBetweenSleep = 2.seconds)(f)
+```
+
+We have also promoted the SFTP file descriptor credentials from a tuple to a dedicated data type:
+
+```scala
+case class Credentials(host: String, port: String, auth: Either[Identity, String])
+```
+
+Additionally, the `Decoder` for [Squants](https://github.com/typelevel/squants)'s `Currency` was updated to become case
+insensitive, meaning that, for example, both `"usd"` and `"USD"` now decode to the same `USD` currency.
```

Comment:
Perhaps a bit hard on the eye, too many commas. I would suggest something like:

```suggestion
insensitive. This means that, for example, both `"usd"` and `"USD"` now decode to the same `USD` currency.
```

Reply:
Yes. I agree.